extensions/wikia/PartnerFeed/PartnerFeed.body.php

```diff
@@ -129,18 +129,23 @@ private function FeedRecentBlogComments ( $format ){
 					if ( ( $maxNumberOfBlogComments-- ) == 0){
 						break;
 					}
-					$tmpArticleComment = $commentPage['level1']->getData();
-
-					$feedArray[] = array(
-						'title' => '',
-						'description' => $tmpArticleComment['text'],
-						'url' => $oTitle->getFullURL(),
-						'date' => $commentPage['level1']->mFirstRevision->getTimestamp(),
-						'author' => $tmpArticleComment['author']->getName(),
-						'otherTags' => array(
-							'image' => AvatarService::getAvatarUrl( $commentPage['level1']->mUser->getName(), $userAvatarSize )
-						)
-					);
+					//make sure all data is loaded
+					if ( $commentPage[ 'level1' ]->load() ) {
+						$tmpArticleComment = $commentPage[ 'level1' ]->getData();
+
+						$feedArray[ ] = array(
+							'title' => '',
+							'description' => $tmpArticleComment[ 'text' ],
+							'url' => $oTitle->getFullURL(),
+							'date' => $commentPage[ 'level1' ]->mFirstRevision->getTimestamp(),
+							'author' => $tmpArticleComment[ 'author' ]->getName(),
+							'otherTags' => array(
+								'image' => AvatarService::getAvatarUrl( $commentPage[ 'level1' ]->mUser->getName(), $userAvatarSize )
+							)
+						);
+					} else {
+						\Wikia\Logger\WikiaLogger::instance()->error('Comment page loading failed', $commentPage);
+					}
```

Comment:
Thanks for the fix!

Minor comment: store ``$commentPage[ 'level1' ]`` in a variable and use it in this block.